metrics/metrics.go

```diff
@@ -134,6 +134,7 @@ func RegisterMetrics() {
 	prometheus.MustRegister(PanicCounter)
 	prometheus.MustRegister(PlanCacheCounter)
 	prometheus.MustRegister(PlanCacheMissCounter)
+	prometheus.MustRegister(InstancePlanCacheMemoryUsage)
```

Comment:
How about using `PlanCache` as the prefix: `PlanCacheInstanceMemoryUsage`?

Reply:
updated